net/yate/Makefile

```diff
@@ -8,15 +8,15 @@
 
 include $(TOPDIR)/rules.mk
 
-RELEASEVER:=5.5.0
+RELEASEVER:=6.0.0
 
 PKG_NAME:=yate
 PKG_VERSION:=$(RELEASEVER)-1
-PKG_RELEASE:=2
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=http://yate.null.ro/tarballs/yate5/
-PKG_MD5SUM:=c433b3df08a5993e16fadaac279e39a0
+PKG_SOURCE_URL:=http://yate.null.ro/tarballs/yate6/
+PKG_MD5SUM:=a21c232f3b7d4abe77d0d4c8cadd6f1e
```

Comment:
@robho, please, change the PKG_MD5SUM variable to PKG_HASH with the SHA256 checksum of the yate archive.